tdvt/samples/connector-tests/propertiesbuilder.xml

```diff
@@ -0,0 +1,8 @@
+<!-- The properties builder test is only used for jdbc based connectors. -->
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>
+    <testCase>
+        <connection class='postgres_jdbc' dbname='TestV1' odbc-native-protocol='' one-time-sql='' port='5432' server='localhost' username='test' password='password' />
+        <expected />
```

Comment:
What is this testing? Shouldn't it have these things in the [expected](https://github.com/tableau/connector-plugin-sdk/blob/master/samples/plugins/postgres_jdbc/connectionProperties.js)?

Reply:
Actually when i ran the command 

tableau-1.3\build\Release-x64\tabquerycli.exe  --conn-test="propertiesBuilder" --conn-test-file="D:\dev\setup.propertiesbuilder.xml" -DConnectPluginsPath=D:\dev\near_jai\tableau-tests\tdvt\plugins

with the expected as empty, the test passed and when i looked at some examples for properties builder tests i noticed that there are tests with empty expected eg : 

https://opengrok.prod.tableautools.com/source/xref/teams_near/testresources/connectors/tabmixins/DataDrivenDriverResolverTest/teradata/setup.propertiesbuilder.xml

Reply:
Nevermind, i realize now that , that example is outdates and it does not use the connectionproperties.js file.  Will update the expected file as rerunning the test with the postgres_jdbc example in the connector packager gives the expected results

Reply:
Fixed in the latest revision